bliss/__init__.py

```diff
@@ -0,0 +1,10 @@
+# flake8: noqa
```

Comment:
Does this disable all flake8 checks? If so, could you instead just disable the one that's problematic?

Reply:
Is there anywhere else this command could go? I'm not sure where, but I'm concerned that new developers on the project won't find it if it's here, and therefore won't know about it.

Reply:
There isn't a way to disable a specific flake8 check for the whole file in the file itself - I think you need to modify the command line flags for that. If we pull everything but an import out of the file though (see response to other comment) then there wouldn't be a risk of missing something by suppressing the check.

Reply:
Got it, seems fine then